buildpack.toml

```diff
@@ -38,23 +37,23 @@ api = "0.2"
     deprecation_date = "2021-02-25T00:00:00Z"
     id = "go"
     name = "Go"
-    sha256 = "a9579c40c6ecca06f7c40ff7563b259cfe33bd20c0611d6c48a76c3811bff8c2"
-    source = "https://dl.google.com/go/go1.14.3.src.tar.gz"
-    source_sha256 = "93023778d4d1797b7bc6a53e86c3a9b150c923953225f8a48a2d5fabc971af56"
+    sha256 = "73fa2a105323f57e3d13fe710d856e1438861aff561f5270d8d442ab004be857"
+    source = "https://dl.google.com/go/go1.14.4.src.tar.gz"
+    source_sha256 = "7011af3bbc2ac108d1b82ea8abb87b2e63f78844f0259be20cde4d42c5c40584"
     stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "org.cloudfoundry.stacks.cflinuxfs3"]
-    uri = "https://buildpacks.cloudfoundry.org/dependencies/go/go_1.14.3_linux_x64_cflinuxfs3_a9579c40.tgz"
-    version = "1.14.3"
+    uri = "https://buildpacks.cloudfoundry.org/dependencies/go/go_1.14.4_linux_x64_cflinuxfs3_73fa2a10.tgz"
+    version = "1.14.4"
 
   [[metadata.dependencies]]
     deprecation_date = "2021-02-25T00:00:00Z"
     id = "go"
     name = "Go"
-    sha256 = "73fa2a105323f57e3d13fe710d856e1438861aff561f5270d8d442ab004be857"
-    source = "https://dl.google.com/go/go1.14.4.src.tar.gz"
-    source_sha256 = "7011af3bbc2ac108d1b82ea8abb87b2e63f78844f0259be20cde4d42c5c40584"
+    sha256 = "3f7111b6fa76d86421563a1cf768a90e1c8d3d5986926a72601d7e7e81884889"
+    source = "/dl/go1.14.5.src.tar.gz"
```

Comment:
@paketo-buildpacks/core-dependencies this URL doesn't look right. Is this expected?

Reply:
Fixed now. Thanks for pointing this out!